spec/classes/datadog_agent_integrations_nginx_spec.rb

```diff
@@ -43,11 +43,25 @@
               'nginx_status_url' => 'http://foo.bar:1941/check',
               'tags' => %w{foo bar baz}
             }
+          ],
+          logs: [
+            {
+              'type' => 'file',
+              'path' => '/var/log/nginx/access.log',
+              'service' => 'my.nginx.service',
+              'source' => 'nginx',
+              'sourcecategory' => 'http_web_access'
+            }
```

Comment:
The log collection for nginx also takes `error.log` into consideration too. Would you want to include it too then (https://github.com/DataDog/integrations-core/blob/7e794127f1312d6830880f35bee54ae2668f25f6/nginx/datadog_checks/nginx/data/conf.yaml.example#L85)?

Reply:
This would be nice, yup :)

Reply:
@jadams-av you can resolve this since you included the error log file, thanks for doing that.